contrib/terraform/gcp/modules/kubernetes-cluster/main.tf

```diff
@@ -231,6 +231,11 @@ resource "google_compute_instance" "master" {
   lifecycle {
     ignore_changes = [attached_disk]
   }
+
+  scheduling {
+    preemptible = var.master_preemptible
+    automatic_restart = !var.worker_preemptible
```

Comment:
Why here needs to refer the worker option instead of the master?

Reply:
I copy-pasted it. Fixed.